frontend/lib/identification/connection_failed_dialog.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:ehrenamtskarte/identification/info_dialog.dart';
+import 'package:flutter/material.dart';
+
+class ConnectionFailedDialog extends StatelessWidget {
+  final String reason;
+
+  const ConnectionFailedDialog({super.key, required this.reason});
+
+  @override
+  Widget build(BuildContext context) {
+    return InfoDialog(
+      title: "Keine Verbindung möglich",
```

Comment:
Maybe add an explanation here:

```suggestion
      title: "Keine Verbindung möglich. Dies bedeutet nicht dass der QR Code ungültig ist. Versuchen Sie erneut die Karte zu scannen.",
```

Reply:
There's an explanation in Text(reason) below.